### ISSUE TITLE
Further error after fixing "ProgressBar object is not an iterator" error

### ISSUE THREAD

**Issue by chrisisbd** [saimn/sigal#372]
I was running sigal 1.4.0 successfully on my xubuntu systems until recently when I upgraded to xubuntu 19.04.  Presumably this is what provoked the "ProgressBar object is not an iterator" error.  So, I searched for the error and got to issue #342 here which (seems to) say that the issue is fixed in version 1.4.1.

I have upgraded to version 1.4.1 and now I get:
```
Collecting albums \Traceback (most recent call last):
  File "/usr/local/bin/sigal", line 10, in <module>
    sys.exit(main())
  File "/usr/lib/python2.7/dist-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/usr/lib/python2.7/dist-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/usr/lib/python2.7/dist-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/lib/python2.7/dist-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/lib/python2.7/dist-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/sigal/__init__.py", line 141, in build
    gal = Gallery(settings, ncpu=ncpu)
  File "/usr/local/lib/python2.7/dist-packages/sigal/gallery.py", line 582, in __init__
    album = Album(relpath, settings, dirs, files, self)
  File "/usr/local/lib/python2.7/dist-packages/sigal/gallery.py", line 277, in __init__
    media = Image(f, self.path, settings)
  File "/usr/local/lib/python2.7/dist-packages/sigal/gallery.py", line 86, in __init__
    self._get_metadata()
  File "/usr/local/lib/python2.7/dist-packages/sigal/gallery.py", line 178, in _get_metadata
    iptc_data = get_iptc_data(self.src_path)
  File "/usr/local/lib/python2.7/dist-packages/sigal/image.py", line 254, in get_iptc_data
    iptc_data["description"] = raw_iptc[(2, 120)].decode('utf-8')
AttributeError: 'NoneType' object has no attribute 'decode'
```
Is this another dependency error or what?  

I also notice that there seems to be a sigal version 2.0 but I can't find any trace of it on github, does it exist or not?

**Comment by saimn** [comment]
Hi,

> Presumably this is what provoked the "ProgressBar object is not an iterator" error. So, I searched for the error and got to issue #342 here which (seems to) say that the issue is fixed in version 1.4.1.

Yes indeed, this issue was fixed in version 1.4.1, but then you have another issue with IPTC data containing malformed data. This was fixed in version 2.0 (#355, 818ec7292fe88d6650e2898e16f160e05d4d2386), but version 2.0 now requires Python 3.5 or newer.

**Comment by chrisisbd** [comment]
OK, thanks.  I had quite a bit of difficulty installing version 2.0.  I had to uninstall the 1.4.1 dependencies which is non-trivial because there's no way to remove them using pip.  I had to install pip-autoremove to uninstall them.  Then, doing a normal install of sigal using pip got me version 2 and all is well again.  :-)

It might be worth a note in the installation instructions about having to remove all the 'bits' when upgrading (major versions only, maybe).
